.travis.yml

```diff
@@ -19,6 +19,8 @@ addons:
 
 # We use matrix include so that xspec builds (which are more complicated and long) are started first
 matrix:
+  allow_failures:
+      - python: "3.5"
```

Comment:
This should be exactly the same as the build you want to allow to fail, so here I guess:
```
     - env: XSPECVER="12.9.0i" FITS="astropy" INSTALL_TYPE=develop TEST=submodule MATPLOTLIBVER=1.5
       python: "3.5"
       sudo: required
       dist: trusty
```